### ISSUE TITLE
App crashes right after extensions download

### ISSUE THREAD

**Issue by dickyharper** [scb261/TachiyomiXZM#1]
### Tachiyomi version

App version: xzm-1.2.2 (standard, f7935d7bf, 12, 2022-07-09T07:25Z) 

### Android version

Android version: 5.0.1 (SDK 21) 

### Device

Android build ID: LRX22C.I9500XXUHPJ3 Device brand: samsung Device manufacturer: samsung Device name: ja3g Device model: GT-I9500 Device product name: ja3gxx

### Steps to reproduce

1. Go to the extensions download section
2. Download an extension
3. The app crashes before installing

### Expected behavior

Expected behavior is the app not crashing. 😊

### Actual behavior

---

### Crash logs

[xzm.txt](https://github.com/scb261/TachiyomiXZM/files/9260658/xzm.txt)

### Other details

---

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose). Note: tachiyomiorg/tachiyomi-extensions still supports Android 5.x
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.2.2](https://github.com/scb261/tachiyomixzm/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by scb261** [comment]
It looks like something is wrong with your storage. Try to remove the app and install it on main storage instead of sdcard

**Comment by dickyharper** [comment]
I'm closing Issue. Because I don't use sdcard in my phone. I looked at the log I sent. I saw the `Failed to find mounted volume for /storage/extSdCard/Android/data/` line in an app that started to crash out of nowhere. I think there is something wrong with my phone. The problem does not prevent me from installing and using the extension. Maybe I can try some older versions.

Thank you so much for continuing support for old phones.
